js.js

```diff
@@ -26,3 +26,133 @@ list.forEach((e) => {
+  },
+  {
+    name: 'Multi-Post Stories',
+    image: './img2/Snapshoot Portfolio desk2.png',
+    list: ['Canopy', 'Back End Dev', '2015'],
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+    technologies2: ['Ruby', 'Bootstrap'],
+    liveLink: 'https://github.com/MohamedRoshdy2021',
+    sourceLink: 'https://example.com/source',
+    popuptext2: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industryLorem Ipsum has been the industrys <br></br> standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. ',
+    popuptext: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+  },
+  {
+    name: 'Tonic',
+    list: ['Canopy', 'Back End Dev', '2015'],
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: './img2/Snapshoot Portfolio desk3.png',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+    technologies2: ['Ruby', 'Bootstrap'],
+    liveLink: 'https://github.com/MohamedRoshdy2021',
+    sourceLink: 'https://example.com/source',
+    popuptext: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    popuptext2: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industryLorem Ipsum has been the industrys <br></br> standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry.  ',
+  },
+  {
+    name: 'Multi-Post Stories',
+    image: './img2/Snapshoot Portfolio desk4.JPG',
```

Comment:
- [ ] Kindly correct the path of the image for a better UI. 👇🏽 

![image](https://github.com/MohamedRoshdy2021/mobile-first-project/assets/58642949/942708cf-e69a-4260-88c8-c3df0a720821)